docs/_summary.md

```diff
@@ -55,6 +55,7 @@
   * [Basic Keycodes](keycodes_basic.md)
   * [US ANSI Shifted Keys](keycodes_us_ansi_shifted.md)
   * [Quantum Keycodes](quantum_keycodes.md)
+  * [ADC Driver](feature_adc_driver.md)
```

Comment:
I think this would also make more sense in the "makers and modders" section, along with the i2c driver info and the gpio cmmands.

Reply:
Done.